docs/automation/step-argument-conversions.md

```diff
@@ -18,7 +18,7 @@ A step argument transformation is used to convert an argument if:
 * The regular expression (if specified) matches the original (string) argument
 
 ```{note}
-If multiple matching transformation are available, a warning is output in the trace and the first transformation is used.
+If multiple matching transformations are available and no order is specified, a warning is output in the trace and the first transformation is used. If the order is specified, the transformation with the lowest order value is used.
```

Comment:
I would add a note before this with the default order number we use if not specified (I think we should include that into the docs anyway).

After that, we can simply say something like `If multiple transformations matching with the lowest order number`

Reply:
Thank you for your comment! I've updated the documentation accordingly. Please let me know if any further changes are needed.